tools/gen_usb_descriptor.py

```diff
@@ -21,6 +21,7 @@
                     help='length needed for the serial number in digits')
 parser.add_argument('--output_c_file', type=argparse.FileType('w'), required=True)
 parser.add_argument('--output_h_file', type=argparse.FileType('w'), required=True)
+parser.add_argument('--reduced_endpoint_mode', nargs='?',const=0, type=int)
```

Comment:
Later we can specify specific endpoints to add (or delete).

Reply:
Agreed, see replies to Scotts comments below